available-connectors.json

```diff
@@ -29,7 +29,7 @@
     },
     "supportedByStrongLoop": true
   },
-  {
+  "object-storage": {
```

Comment:
prefix `ibm-`?

Reply:
i thought it was verbose to have `ibm-` in front, but i'll introduce it back since it might not be clear enough